deepreg/model/loss/util.py

```diff
@@ -0,0 +1,10 @@
+import tensorflow as tf
+
+
+class NegativeLossMixin(tf.keras.losses.Loss):
```

Comment:
do we really need a mixin class? it hurts readability to me.

Reply:
Yes we need. As user you do not need to care about mixin.